programs/axelar-solana-gateway/src/processor/approve_message.rs

```diff
@@ -98,13 +99,6 @@ impl Processor {
             return Err(GatewayError::SigningSessionNotValid.into());
         }
 
-        // Check: message signing verifier set matches the verification session verifier set
-        if merkleised_message.leaf.signing_verifier_set
-            != session.signature_verification.signing_verifier_set_hash
-        {
-            return Err(GatewayError::InvalidVerificationSessionPDA.into());
-        }
-
```

Comment:
There is not a concept of "the expected signing verifier set" anymore. Any active/valid verifier set can sign the payload.

We still prevent untrusted verifier sets from verifying their signatures at these two other instructions:
- `process_initialize_payload_verification_session`
- `process_verify_signature`

That means we can trust any valid signature verification session account received at this point (the `process_approve_message` instruction processor)